### ISSUE TITLE
Search box overrides Home and End key

### ISSUE THREAD

**Issue by Nickwiz** [mdn/content#16995]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event

### What specific section or headline is this issue about?

Search box

### What information was incorrect, unhelpful, or incomplete?

The search box has bound `Home` and `End` to jump to start / end of suggestion list. 

This interfere if one want to jump in the *text* one is writing. I.e.

```
some text| here
         |
         +---- cursor here
```

By pressing:
* `Home`: Cursor is not moved to start but one navigates the start of suggestion list.
* `Shift` + `Home`: One do not select all text before cursor, but navigates to start of suggestion list.

Further if there is one or no suggestions it appears as if the `Home` and `End` keys are not working as nothing happens.

Believe one should do a check for `Shift` and `Ctrl` when `Home` and `End` is pressed.

* `Home`: Start of suggestion list
* `End`: End of suggestion list
* `Ctrl` + `Home`: Jump to start of input (Default behavior)
* `Ctrl` + `End`: Jump to end of input (Default behavior)
* (`Ctrl` +)<sup>1</sup> `Shift` + `Home`: Select text from cursor to start (Default behavior)
* (`Ctrl` +)<sup>1</sup> `Shift` + `End`: Select text from cursor to end (Default behavior)

<sup>1: `Ctrl` not needed as it is not a textarea but behavior is the same with / without</sup>

In other words, *"simply"* not navigate the suggestion list if either modifier is pressed and let the browser do it's default action.

One could of course also turn it around and say:

* `Ctrl` + `Home`: Jump to start of suggestion list
* `Ctrl` + `End`: Jump to end of suggestion list

And leave `Home` and `End` as their defaults - but think this would perhaps be more confusing.

### What did you expect to see?

When pressing `Shift` + `Home` for the text to be selected.

### Do you have anything more you want to share?

Not sure if this is the correct place to submit this bug / suggestion, but closest I found.
